src/main/service/manualUploadService.ts

```diff
@@ -72,8 +102,9 @@ export class ManualUploadService {
                 language,
                 languageFile
             ),
+            classificationError: formValues['classification'] ? null : 'true',
         };
-        if (!fields.courtError && !fields.contentDateError && !fields.displayDateError) {
+        if (!fields.courtError && !fields.contentDateError && !fields.displayDateError && !fields.classificationError) {
```

Comment:
Lots of negative conditions here. Is that better to do the following so it will stop evaluating as soon as if find an error?
if (fields.courtError || fields.contentDateError || fields.displayDateError || fields.classificationError) {
    return fields;
}
return null;

Reply:
Good spot. I've updated this now.